.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish packages
+
+on:
+    push:
+        branches:
+          - 'releases/**'
+        tags:
+          - v3
+          - v3.*
+
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout repo
+            uses: actions/checkout@v2
+          - name: Set up Python 3.7
+            uses: actions/setup-python@v1
+            with:
+                python-version: 3.7
+          - name: Install dependencies
+            run: |
+                python -m pip install --upgrade pip poetry==1.0
+          - name: Build wheel
+            run: |
+                poetry build
```

Comment:
What about additional step, which will check that version from pyproject.toml is equal to git tag?

Reply:
Sounds good
And `__version__` from `aiogram/__init__.py` is also should be validated

Reply:
Should we use a tool like `bump2version` or `dephell.versioning` for reducing manual work?

Reply:
@b0g3r check and.. what's next?

Reply:
If it's not we have to stop and mark the action as failed I think

It's like 'oh uh you forgot to bump version in pyproject.toml and __init__.py would you please bump it in before releasing?'

Reply:
In such cases re-pushing the git tag is dirty but acceptable workaround